sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 2.0.0-beta.3 (Unreleased)
+## 2.0.0-beta.3 (2021-05-11)
```

Comment:
Make this 05-12, so that we can test today and click the release button tomorrow 🌞